UI/NativeApp.cpp

```diff
@@ -1406,9 +1408,6 @@ void NativeShutdown() {
 
 	g_Discord.Shutdown();
 
-	delete logger;
```

Comment:
Hm, but we still new in init.  Should we check if `!logger` there?

-[Unknown]

Reply:
Hm I removed this intending to change how LogManager deals with their lifetimes but then backed out and forgot this. Nice catch.